libraries/chain/include/eosio/chain/db_context.hpp

```diff
@@ -0,0 +1,194 @@
+#pragma once
+
+#include <eosio/chain/name.hpp>
+#include <memory>
+#include <stdint.h>
+
+namespace chainbase {
+   class database;
+}
+
+namespace eosio { namespace chain {
+
+      class apply_context;
+
+      inline constexpr name dbram_id  = N(eosio.dbram);
```

Comment:
Is this being defined in multiple translation units?  Otherwise, I don't think `inline` is necessary as the compiler will make that decision on its own regardless.  Same comment for line below.

Reply:
removed inline and removed un-needed disk variable